src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -260,3 +261,102 @@ by setting the parameter `elasticsearch.index.mailbox.name` to the name of your
 re-creates index upon restart
 
 _Note_: keep in mind that reindexing can be a very long operation depending on the volume of mails you have stored.
+
+## Setting Cassandra user permissions
+
+With a Cassandra server that serving multiple applications including James. 
```

Comment:
s/serving/serves/

and you should not close the sentence has it continues on next line (or maybe I don't understand)